src/lib.rs

```diff
@@ -923,6 +946,48 @@ mod tests {
         }
     }
 
+    mod write {
+        use std::error::Error;
+        use std::fs::*;
+        use std::io::Read;
+        use std::path::Path;
+        use super::RECS;
+        use super::REC_SIZE;
+        use super::super::*;
+
+        #[test]
+        fn should_write_record_to_file() {
```

Comment:
Since `WriteRecordExt` will be implemented for every `T: io::Write` there is no need to reduce one's ssd lifetime by writing record into a file on every test run. It could be written into `Vec<u8>` for testing purpose.